sdk/servicebus/service-bus/test/backupMessageSettlement.spec.ts

```diff
@@ -1,336 +1,336 @@
-      );
-      should.equal(deadLetterMsgsBatch.length, 1, "Unexpected number of messages");
-      should.equal(
-        deadLetterMsgsBatch[0].body,
-        testMessages.body,
-        "MessageBody is different than expected"
-      );
-      should.equal(
-        deadLetterMsgsBatch[0].messageId,
-        testMessages.messageId,
-        "MessageId is different than expected"
-      );
-
-      await deadLetterMsgsBatch[0].complete();
-
-      await testPeekMsgsLength(deadLetterReceiver, 0);
-    } else {
-      const messageBatch = await receiver.receiveMessages(1);
-      await messageBatch[0].complete();
-
-      await testPeekMsgsLength(receiver, 0);
-    }
-  }
-
-  it(
-    noSessionTestClientType + ": deadLetter() moves message to deadletter queue",
-    async function(): Promise<void> {
-      await beforeEachTest(noSessionTestClientType);
-      await testDeadletter();
-    }
-  );
-
-  it(
-    withSessionTestClientType + ": deadLetter() moves message to deadletter queue",
-    async function(): Promise<void> {
-      await beforeEachTest(withSessionTestClientType);
-      await testDeadletter();
-    }
-  );
-
-  async function testRenewLock(): Promise<void> {
-    const testMessages = entityNames.usesSessions
-      ? TestMessage.getSessionSample()
-      : TestMessage.getSample();
-    const msg = await sendReceiveMsg(testMessages);
-    await receiver.close();
-    let errorWasThrown = false;
-    try {
-      const lockedUntilBeforeRenewlock = msg.lockedUntilUtc;
-      const lockedUntilAfterRenewlock = await msg.renewLock();
-      should.equal(
-        lockedUntilAfterRenewlock > lockedUntilBeforeRenewlock!,
-        true,
-        "MessageLock did not get renewed!"
-      );
-      await msg.complete();
-    } catch (err) {
-      should.equal(
-        err.message,
-        `Invalid operation on the message, message lock doesn't exist when dealing with sessions`,
-        "Unexpected error thrown"
-      );
-      errorWasThrown = true;
-    }
-
-    receiver = await serviceBusClient.test.createPeekLockReceiver(entityNames);
-    if (entityNames.usesSessions) {
-      should.equal(errorWasThrown, true, "Error was not thrown for messages with session-id");
-      const msgBatch = await receiver.receiveMessages(1);
-      await msgBatch[0].complete();
-    } else {
-      should.equal(errorWasThrown, false, "Error was thrown for sessions without session-id");
-    }
-    await testPeekMsgsLength(receiver, 0);
-  }
-
-  it(noSessionTestClientType + ": Lock renewal for a message", async function(): Promise<void> {
-    await beforeEachTest(noSessionTestClientType);
-    await testRenewLock();
-  });
-
-  it(withSessionTestClientType + ": Lock renewal for session", async function(): Promise<void> {
-    await beforeEachTest(withSessionTestClientType);
-    await testRenewLock();
-  });
-});
+// // Copyright (c) Microsoft Corporation.
+// // Licensed under the MIT license.
+
+// import chai from "chai";
```

Comment:
Why?

Reply:
Ah, I meant to talk about this in the PR description, but missed it.

Now that we have the settlement methods on the receiver, they cannot be used once the receiver is closed.
What is possible is that the user facing receiver is not closed, but the underlying AMQP link is closed which is when the backup settlement would still apply. I kept it out of scope in my first pass because the changes were not trivial and I wanted this PR to be the first step towards #11926

Reply:
See https://github.com/Azure/azure-sdk-for-js/pull/11962#issuecomment-714184513